### ISSUE TITLE
Don't exit on first key press

### ISSUE THREAD

**Issue by jdumas** [carnager/teiler#41]
Hi,

One thing that I found really annoying with `teiler` is that it exits by default when you press <kbd>Enter</kbd> after launching it. That means if you open `teiler` via dmenu or rofi, then you press <kbd>Enter</kbd> and `teiler` immediately exits, making it basically impossible to use it. I have to be extra very careful to press <kbd>Enter</kbd> only for the tiniest amount of time to be able to run a command =/

I don't if that's something you've experienced too, so I wanted to signal it.

**Comment by carnager** [comment]
I guess I could remove the Exit menu entry and only exit on ESC.

But to be honest, i NEVER use the menu, instead I have hotkeys for most used functions.
e.g. I have Mod+Print for `teiler --quick image area` and Shift+Print for `teiler --togglecast`

**Comment by jdumas** [comment]
Yeah that's what I thought. I should do that as well (esp. since when doing a screencast the menu does show on the screen before stopping the cast).

**Comment by Amooti73** [comment]
Thanks for calling attention to the use of hotkeys.  I am new to teiler and was also experiencing the same problem as [jdumas](https://github.com/jdumas), but the use of a hotkey seems to have solved, at least for the area screenshot.
